specification/security/resource-manager/Microsoft.Security/stable/2018-06-01/security.json

```diff
@@ -0,0 +1,253 @@
+                }, {
+                    "$ref": "#/parameters/SubscriptionId"
+                }, {
+                    "$ref": "#/parameters/PricingName"
+                }, {
+                    "$ref": "#/parameters/Pricing"
+                }],
+                "responses": {
+                    "202": {
+                        "description": "Accepted -- Update request accepted; operation will complete asynchronously."
+                      }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }
+        },
+    "definitions": {
+        "CloudError": {
+            "x-ms-external": true,
+            "type": "object",
+            "description": "Error response structure.",
+            "properties": {
+                "error": {
+                    "x-ms-client-flatten": true,
+                    "description": "Error data",
+                    "$ref": "#/definitions/CloudErrorBody"
+                }
+            }
+        },
+        "CloudErrorBody": {
+            "x-ms-external": true,
+            "type": "object",
+            "description": "Error details.",
+            "properties": {
+                "code": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+                },
+                "message": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "A message describing the error, intended to be suitable for display in a user interface."
+                }
+            }
+        },
+        "PricingList": {
+            "description": "List of pricing configurations response",
+            "required": [
+                "value"
+            ],
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "description": "List of pricing configurations",
+                    "items": {
+                        "$ref": "#/definitions/Pricing"
+                    }
+                }                
+            }
+        },
+        "Pricing": {
+            "type": "object",
+            "description": "Pricing tier will be applied for the scope based on the resource ID",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Pricing data",
+                    "$ref": "#/definitions/PricingProperties"
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"

```

Comment:
this definition is missing (this is what is failing the test)

Reply:
thanks